### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by Anshgrover23**
Fixes: #2216 
## Description

There are some Broken links in readme.md

## Changes

I have changed the broken links attached to **white labelling, CSS Variables for non-React Projects and Rfcs** in readme.md file

## Component Checklist

- [x] Perform Manual Testing in Other Browsers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ Blade is the Design System that powers [Razorpay](https://razorpay.com/).
 
 ## ✨ Features
 - Cross-Platform (Works Natively on [React Web](https://blade.razorpay.com/?path=/docs/guides-installation--page#%EF%B8%8F-installation) and [React Native](https://blade.razorpay.com/?path=/docs/guides-installation--page#react-native-projects))
```
Comment (saurabhdaware) [inline README.md:28]: can you fix these links as well? seems like they are broken too

Comment (Anshgrover23) [reply README.md]: done.

**Review by saurabhdaware — APPROVED** [review]
Thanks for your contribution 🏅

File: README.md [hunk 2]
```diff
@@ -25,11 +25,11 @@ Blade is the Design System that powers [Razorpay](https://razorpay.com/).
 - [@razorpay/blade-old](https://github.com/razorpay/blade-old) (Deprecated, Private)
 
 ## ✨ Features
-- Cross-Platform (Works Natively on [React Web](https://blade.razorpay.com/?path=/docs/guides-installation--page#%EF%B8%8F-installation) and [React Native](https://blade.razorpay.com/?path=/docs/guides-installation--page#react-native-projects))
-- [White Labelling](https://blade.razorpay.com/?path=/docs/guides-theming-theme-playground--page)
-- [CSS Variables for non-React Projects](https://blade.razorpay.com/?path=/docs/tokens-css-variables--page)
+- Cross-Platform (Works Natively on [React Web](https://blade.razorpay.com/?path=/docs/guides-installation--docs) and [React Native](https://blade.razorpay.com/?path=/docs/guides-installation--docs))
+- [White Labelling](https://blade.razorpay.com/?path=/docs/guides-theming-theme-playground--docs)
+- [CSS Variables for non-React Projects](https://blade.razorpay.com/?path=/docs/tokens-typography--docs)
```
Comment (saurabhdaware) [inline README.md:30]:
```suggestion
```

This feature we removed so just removing this point

File: README.md [hunk 3]
```diff
@@ -25,11 +25,11 @@ Blade is the Design System that powers [Razorpay](https://razorpay.com/).
 - [@razorpay/blade-old](https://github.com/razorpay/blade-old) (Deprecated, Private)
 
 ## ✨ Features
-- Cross-Platform (Works Natively on [React Web](https://blade.razorpay.com/?path=/docs/guides-installation--page#%EF%B8%8F-installation) and [React Native](https://blade.razorpay.com/?path=/docs/guides-installation--page#react-native-projects))
-- [White Labelling](https://blade.razorpay.com/?path=/docs/guides-theming-theme-playground--page)
-- [CSS Variables for non-React Projects](https://blade.razorpay.com/?path=/docs/tokens-css-variables--page)
+- Cross-Platform (Works Natively on [React Web](https://blade.razorpay.com/?path=/docs/guides-installation--docs) and [React Native](https://blade.razorpay.com/?path=/docs/guides-installation--docs))
```
Comment (saurabhdaware) [inline README.md:28]:
```suggestion
- Cross-Platform (Works Natively on [React Web and React Native](https://blade.razorpay.com/?path=/docs/guides-installation--docs))
```

Its the same link so combining them

**Review by saurabhdaware — APPROVED** [review]

**Comment by Anshgrover23** [comment]
@saurabhdaware when it will get merged may i know sir?

**Comment by saurabhdaware** [comment]
> @saurabhdaware when it will get merged may i know sir?

Now 🤭 Sorry I was in meeting.

**Comment by Anshgrover23** [comment]
@saurabhdaware no worries and thanks for merging brother.

**Pull request merged**